appengine/MIDI.js

```diff
@@ -0,0 +1 @@
+Subproject commit a8a84257afa70721ae462448048a87301fc1554a
```

Comment:
I don't know. I just followed your instructions to get the MIDI.js file and copied it into this folder.